src/components/Cards/Reviewcards.tsx

```diff
@@ -0,0 +1,113 @@
+import { UseFetch } from "../../customHook/UseFetch";
+import { Review } from "../../types/types";
+
+import { Rating } from "@mui/material";
+import { MDBContainer, MDBTypography } from "mdb-react-ui-kit";
+import { Carousel } from "react-bootstrap";
+import {useEffect} from "react";
+import Person from "../../assets/images/person.png";
+import "./slider.css";
+import "./roomcards.css";
+
+interface Res {
+  data: {
+    Review: Review[];
+  };
+  loading: boolean;
+}
+
+const Reviewcards = ({ featured }) => {
+  const { data, loading }: Res = UseFetch(
+    `reviews${featured}`
```

Comment:
use the ternary operator here and check if featured then use the featured route else empty string